vyxal/elements.py

```diff
@@ -5858,6 +5859,14 @@ def separate_runl_encode(lhs, ctx: Context):
     return lengths
 
 
+@element("ø%", 1)
+def sha256_hash(lhs, ctx):
+    """Element ø%
+    (any) -> hash a using SHA256
```

Comment:
```suggestion
    (str) -> hash a string using SHA256
```
This only works on strings, not any value (I guess you could do `str(lhs).encode()`, but I don't see much point in bothering)

Reply:
It *does* convert to string. `str(lhs).encode()` is in the code already.